### DISCUSSION TITLE
bug in github

### DISCUSSION THREAD

**Discussion by OsamaMohamedAtta** [community/community#111255] [Repositories]
### Select Topic Area

Bug

### Body

i have a proplem when i upload some file by drag and drop in my repo he told me commit failed

**Comment by gil-son** [comment]
Could you share the error message?

**Reply by OsamaMohamedAtta** [reply]
![Screen Shot 2024-03-07 at 12 03 13 PM](https://github.com/community/community/assets/91071617/dbbeb377-5673-40a2-b75e-b81c86c0b5ff)
this is the error message, this error just happened today

**Reply by gil-son** [reply]
Witch kind of files you tried to import?

Maybe it can be a bug. But, consider to clone your repository to your computer and add the files in there

**Comment by Gubna-Tech** [comment]
I have been having this issue all day, fails to upload file(s) and gives me an option to retry.

**Reply by OsamaMohamedAtta** [reply]
yes, me too i think this is a bug in github

**Reply by Tyler-Braxton** [reply]
I have been having the same problem. Some of my files got uploaded but 2 files won't upload.
